modules/freetype/config.py

```diff
@@ -3,4 +3,10 @@ def can_build(env, platform):
 
 
 def configure(env):
-    pass
+    from SCons.Script import BoolVariable, PathVariable, Variables, Help
+
+    opts = Variables()
+    opts.Add(BoolVariable("brotli", "Enable Brotli decompressor for WOFF2 fonts support", True))
+
+    opts.Update(env)
+    Help(opts.GenerateHelpText(env))
```

Comment:
I think updating the opts and help might mess with this hack repeated multiple times in the `SConstruct`: https://github.com/godotengine/godot/blob/2c0c76c415f1b8b7d46f63d12b12ef56c8c85f4d/SConstruct#L307

We should probably solve it properly as it's a hassle.

As a side note, `PathVariable` doesn't seem used?

Reply:
"Mono" module seems to be doing the same, without resetting `env_base["platform"]`.

Maybe modules should have separate function to return options, like platforms.

Reply:
> I think updating the opts and help might mess with this hack repeated multiple times

Does not seem to be any difference, but I have replaced it with the `get_opts()` method, similar to the one used in platforms.